jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js

```diff
@@ -54,8 +54,6 @@ YUI.add('service-overview', function() {
       const props = this.props;
 
       if (!props.displayPlans) {
```

Comment:
Still not sure why the if statement and prop remain. If it's unused/always false why not nuke it?